src/entities.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import json
```

Comment:
add a module docstring before the imports that provides a quick overview of the purpose of this script.  that way when other devs open this file, they can quickly figure out what it's for.